app/lib/katello/concerns/base_template_scope_extensions.rb

```diff
@@ -38,6 +39,10 @@ def host_latest_applicable_rpm_version(host, package)
         host.applicable_rpms.where(name: package).order(:version_sortable).limit(1).pluck(:nvra).first
       end
 
+      def host_installed_packages(host)
+        host.installed_packages.map(&:name)
```

Comment:
I think we should rename this to host_installed_package_names since we are only returning names.